datajoint/autopopulate.py

```diff
@@ -16,28 +17,27 @@ class AutoPopulate(metaclass=abc.ABCMeta):
     """
     AutoPopulate is a mixin class that adds the method populate() to a Relation class.
     Auto-populated relations must inherit from both Relation and AutoPopulate,
-    must define the property populated_from, and must define the callback method _make_tuples.
```

Comment:
MATLAB does not yet have a default `popRel` implemented, so every computed class does have one.

The `poprel` feature needs to be documented fairly well.  Try using 'populated_from` in a sentence.  It is easy to define a `poprel` in the first paragraph and then use it throughout.  From the name alone, no one will understand what `populated_from` actually is, so the name is not that descriptive.  Might as well give it a name and describe it in the tutorial.